### PR TITLE
Update PR process as it has evolved

### PR DESCRIPTION
**Pull request by matks**
Update devdocs as we have removed PrettyCI, added Github Actions and added Issuebot

### CODE REVIEW

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md [hunk 1]
```diff
@@ -13,33 +13,35 @@ When you submit a new Pull Request to the [project repository](https://github.co
...
 
-If PrettyCI states that your pull request has issues, you need to fix the issues by looking at PrettyCI output to understand what needs to be fixed.
+If the Github Action states that your pull request has issues, you need to fix the issues by looking at the Github Action output to understand what needs to be fixed.
 
-The second tool is [Travis](https://travis-ci.org/). Travis is a Continous Integration system that will look at the Pull Request and run several checks, be it code-style checkers, format checkers or automated tests, and provide you the result in the Pull Request. If something is wrong it will block the PR from being merged. This is a standard approach to ensure that new contributions in a codebase do not break existing features and behaviors.
+The second tool is a [ESLint](https://eslint.org/) [GitHub Action](https://github.com/features/actions). This one checks whether the Javascript syntax and code-style is correct. Likewise, it will block the PR from being merged if there is a syntax or code-style issue.
```
Comment (PierreRambaud) [inline src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md:25]: Could you do the same for stylelint? (please :heart: )

**Comment by PierreRambaud** [comment]
Great initiative @matks !

File: src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md]: Done in https://github.com/PrestaShop/docs/pull/526/commits/cbb6823bf9e9bef9ad9a63490de0273daa189809

**Comment by matks** [comment]
Thank you for review

**Pull request merged**